### PR TITLE
Move rest of the non-sensitive configuration to appsettings

### PR DESCRIPTION
**Pull request by acn-dgopa**
Moved the following to appsettings

- cacheconfig
- keyvaultsettings
- clientsettings

## Related Issue(s)
- #308 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json [hunk 1]
```diff
@@ -10,5 +10,18 @@
   "GeneralSettings": {
     "FrontendBaseUrl": "https://am.ui.at21.altinn.cloud",
     "Hostname": "at21.altinn.cloud"
+  },
+  "KeyVaultSettings": {
+    "SecretUri": "https://altinn-at21-am-kv.vault.azure.net/"
```
Comment (jonkjetiloye) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json:15]: Why do we need two KeyVault URIs?

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json [hunk 2]
```diff
@@ -10,5 +10,18 @@
...
+  },
+  "ClientSettings": {
```
Comment (jonkjetiloye) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json:17]: For ClientSettings, 3/4 values are common and could be setup in the non-env appsettings.json and the fourth is the same as KeyVaultSettings URI

Comment (acn-dgopa) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json]: Moved them to appsettings

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json (hunk 1 above)
Comment (acn-dgopa) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json:15]: Removed the clientsettings config and reusing the keyvaultsettings uri

**Review by jonkjetiloye — APPROVED** [review]

**Pull request merged**